client/src/session.rs

```diff
@@ -37,9 +37,10 @@ pub fn get_nexus_address() -> String {
     format!("127.0.0.1:{}", port)
 }
 
-#[pymethods]
+#[cfg_eval]
```

Comment:
We'll want to get rid of this as it requires a nightly build of rust to work.  I needed it to support the nested #[new] and #[getter] tags but we can just set those up in the py wrapper dylib.